lang/fr.json

```diff
@@ -140,7 +140,7 @@
 "Settings":"Paramètres",
 "Change settings.":"Modifier les paramètres.",
 "Default boot":"Démarrage par défaut",
-"Force rxTools UI":"Forcer le démarrage de l'interphace graphique (GUI)",
+"Force rxTools UI":"Forcer le démarrage de l'interface graphique (GUI)",
```

Comment:
Such long will take 2 or more strings in settings menu page, please keep as short as possible.